docs/guides/bridge.md

```diff
@@ -1,16 +1,14 @@
-# kBTC/interBTC Bridge Guide (Testnet)
+# kBTC/interBTC Bridge Guide
 
-?> Warning: These guides are for Testnet only! Kintsugi and Interlay bridge guides are WIP. The process is the same but specific naming might have changed. 
+## Issue
 
-## Issue (Testnet)
-
-interBTC allows you to receive a representation of BTC to be used any way you see fit in the Polkadot ecosystem.
+kBTC or interBTC allows you to receive a representation of BTC to be used any way you see fit in the Polkadot ecosystem.
 To get you started, follow this guide.
 
 At the end of this guide you will have:
 
 - [x] Locked BTC with a collateralized Vault
-- [x] Issued your first interBTC on the interBTC app
+- [x] Issued your first kBTC/interBTC on the kBTC/interBTC app
 
 ### Video Guide (OLD UI - New Guide is WIP)
```

Comment:
Can you please remove the video guide for now? We can add it later again once we updated it.